### DISCUSSION TITLE
扫描件pdf版面恢复成docx文件时，会将整个页面输出为图片，能否恢复为可编辑的docx呢，测试图片和输出在附件。

### DISCUSSION THREAD

**Discussion by Gavin-debug** [PaddlePaddle/PaddleOCR#13058] [Q&A]
使用的脚本：
python3 ppstructure/predict_system.py --image_dir=images/test/input
[test.zip](https://github.com/user-attachments/files/15816574/test.zip)
.pdf --recovery=True --use_pdf2docx_api=True --output ./output

**Comment by SWHL** [comment]
这个估计是版本分析模型将当前页面识别为图片了，整个放到docx里了。
你试试官方给的图像能不能行。参考[link](https://github.com/PaddlePaddle/PaddleOCR/blob/main/ppstructure/docs/quickstart.md#223-%E7%89%88%E9%9D%A2%E5%88%86%E6%9E%90)

**Reply by Gavin-debug** [reply]
@SWHL 官方图像能输出图像中的图片和图像中的其他模块，所以pdf转word的功能现在还不支持直接识别扫描件里的文字吗
<img width="1301" alt="image" src="https://github.com/PaddlePaddle/PaddleOCR/assets/82193319/4bf62a08-e8e9-4a95-aa17-47c01ea43025">
<img width="992" alt="image" src="https://github.com/PaddlePaddle/PaddleOCR/assets/82193319/dc6ae9d5-6e9c-42d8-b2cc-dd8901a58dc1">

**Reply by Gavin-debug** [reply]
ocr功能是能识别的，但是转word好像不太行，转完还是以一整张图片的形式保存的，而不是识别其中的文字，变成可编辑的word

**Reply by SWHL** [reply]
你这种图像不行，因为这个structure中最关键的是版面分析模型，版面分析模型框选区域对了，才有可能后续识别对。
你的这个图，版面分析模型训练的时候就没见过，估计就把整个页面作为一整个图像了。

**Comment by swmp** [comment]
我也遇到这个问题了，扫描版本的文件很容易被原封不动保存为图片，例如doc中的图片 doc/imgs_en/img_12.jpg 
paddleocr --image_dir=ppocr_img/ppocr_img/imgs_en/img_12.jpg --type=structure --recovery=true --lang="en"
如果把 --lang="en"去掉，则可以保存为文本的docx文件，但是文字识别率会变低
[output.zip](https://github.com/user-attachments/files/16758886/output.zip)
![img_12](https://github.com/user-attachments/assets/a5448460-79d1-4750-a7d3-5451882da1da)

**Reply by SWHL** [reply]
这个因为版面分析模型不行，需要自己微调一下才好使

**Reply by swmp** [reply]
> 这个因为版面分析模型不行，需要自己微调一下才好使

从res_0.txt看识别的成功率上还可以接受，保存为docx时出了bug
